### DISCUSSION TITLE
Is it possible to select the pane under the cursor when focusing a wezterm window on a mac?

### DISCUSSION THREAD

**Discussion by mivok** [wezterm/wezterm#880] [Q&A] [answered]
Hi.

I currently have wezterm running with multiple panes in a single window. If I have another window focused, and click on the wezterm window, but over an inactive pane, I need to click a second time in order to focus the pane as well. Is it possible to have the pane be immediately focused on the first click that activates the window?

I do have `pane_focus_follows_mouse = true`, which _almost_ does what I want. Unfortunately, if I click the window to focus it, and don't also move the mouse after clicking, then the pane under the cursor never actually gets focused. I also have `swallow_mouse_click_on_pane_focus = false` (or rather I have it left at the default), but that doesn't appear to be what I'm looking for either.

**Comment by wez** [comment, marked as answer]
Hey @mivok, it's been a while, hope you're well!
This sounds like a legit bug, so I opened: https://github.com/wez/wezterm/issues/881

**Reply by mivok** [reply]
Thanks, and it has indeed been a while. When I saw you'd wrote a terminal emulator I had to try it and so far it's amazing. If you're ever back in MD let me know - we should meet up.
